src/test/java/seedu/address/ui/UiPartTest.java

```diff
@@ -27,49 +27,49 @@
     public TemporaryFolder testFolder = new TemporaryFolder();
 
     @Test
-    public void constructor_nullFileUrl_throwsNullPointerException() {
+    public void constructorNullFileUrlThrowsNullPointerException() {
         thrown.expect(NullPointerException.class);
         new TestUiPart<Object>((URL) null);
     }
 
     @Test
-    public void constructor_missingFileUrl_throwsAssertionError() throws Exception {
+    public void constructorMissingFileUrlThrowsAssertionError() throws Exception {
         URL missingFileUrl = new URL(testFolder.getRoot().toURI().toURL(), MISSING_FILE_PATH);
         thrown.expect(AssertionError.class);
         new TestUiPart<Object>(missingFileUrl);
     }
 
 
-    //    @Test
-    //    public void constructor_invalidFileUrl_throwsAssertionError() {
-    //        URL invalidFileUrl = getTestFileUrl(INVALID_FILE_PATH);
-    //        thrown.expect(AssertionError.class);
-    //        new TestUiPart<Object>(invalidFileUrl);
-    //    }
-    //
-    //    @Test
-    //    public void constructor_validFileUrl_loadsFile() {
-    //        URL validFileUrl = getTestFileUrl(VALID_FILE_PATH);
-    //        assertEquals(VALID_FILE_ROOT, new TestUiPart<TestFxmlObject>(validFileUrl).getRoot());
-    //    }
+    @Test
+    public void constructorInvalidFileUrlThrowsAssertionError() {
+        URL invalidFileUrl = getTestFileUrl(INVALID_FILE_PATH);
+        thrown.expect(AssertionError.class);
+        new TestUiPart<Object>(invalidFileUrl);
+    }
 
     @Test
-    public void constructor_nullFileName_throwsNullPointerException() {
+    public void constructorValidFileUrlLoadsFile() {
+        URL validFileUrl = getTestFileUrl(VALID_FILE_PATH);
+        assertEquals(VALID_FILE_ROOT, new TestUiPart<TestFxmlObject>(validFileUrl).getRoot());
+    }
+
+    @Test
+    public void constructorNullFileNameThrowsNullPointerException() {
         thrown.expect(NullPointerException.class);
         new TestUiPart<Object>((String) null);
     }
 
     @Test
-    public void constructor_missingFileName_throwsNullPointerException() {
+    public void constructorMissingFileNameThrowsNullPointerException() {
         thrown.expect(NullPointerException.class);
         new TestUiPart<Object>(MISSING_FILE_PATH);
     }
 
-    //    @Test
-    //    public void constructor_invalidFileName_throwsAssertionError() {
-    //        thrown.expect(AssertionError.class);
-    //        new TestUiPart<Object>(INVALID_FILE_PATH);
-    //    }
+    @Test
+    public void constructorInvalidFileNameThrowsAssertionError() {
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation:exclamation: One or more of these problems noticed: 
1. extra/missing spaces
2. brace style problem
3. missing braces
4. name doesn't follow naming convention
5. line too long

Reply:
The name might not satisfy codacy if we change it to include underscores though. Shall we standardise and change everything to have no underscores to appease codacy?

Reply:
you can turn off that rule in codacy.

Reply:
Oh really! I didn't know that :P